packages/dev-utils/src/test/generate-image.ts

```diff
@@ -0,0 +1,19 @@
+import { generateImage as generateImageCallback } from 'js-image-generator'
+
+/**
+ * Returns Buffer of a generated random noise jpeg image with the specified width and height.
+ */
+export async function generateImage(width: number, height: number): Promise<Buffer> {
```

Comment:
To avoid adding fixture style image to repo, this will generate images on-demand (random noise, so they don't mean anything, but can be used to assert resizing)

In this PR this is used in ~unit tests for ImageHandler AND e2e for vite plugin

Additionally minor adjustments to `Fixture` to allow return of this image to feed into `Fixture.withFile` builder